components/Footer.tsx

```diff
@@ -0,0 +1,30 @@
+import '@fontsource/roboto';
+import { Grid, Typography } from '@mui/material';
+
+export function Footer() {
+  return (
+    <footer>
+      <Grid
+        container
+        direction='column'
+        alignItems='center'
+        justifyContent='center'
+        bgcolor='#1976D2'
+        spacing={0}
+        padding={0}
+        height={158}
+      >
+        <Grid
+          style={{ textAlign: 'center' }}
+          color='#ffffff'
+        >
+          <div style={{ alignItems: 'center', display: 'flex', gap: 4 }}>
```

Comment:
Styles should be placed in .scss file not inline in JavaScript Code

Reply:
Design does not use scss files anymore